src/Shared/test/Microsoft.Extensions.Sources.Tests.csproj

```diff
@@ -0,0 +1,32 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFrameworks>$(StandardTestTfms)</TargetFrameworks>
+    <DebugType>portable</DebugType>
+    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <Compile Include="..\src\**\*.cs"
+             Exclude="
+              ..\src\BenchmarkRunner\**\*.cs;
```

Comment:
How gross is this exclusion?

Reply:
i'm open to suggestions if this is too cross. 🤢

Reply:
Is this likely to cause any unexpected test bugs in the future? Or would a "typical failure" be immediately obvious?

Reply:
I don't think this will cause unexpected test bugs. These particular files caused compiler errors when included, which is why the exclusion was necessary. Compiler errors are immediately obvious, so any changes to this should be quickly identified.